src/sentry/static/sentry/app/components/forms/inputField.jsx

```diff
@@ -7,9 +7,11 @@ export default class InputField extends FormField {
   constructor(props) {
     super(props);
 
-    this.state.value = (
-      props.value !== '' ? props.value : (props.defaultValue || '')
-    );
+    this.state.value = this.valueFromProps(props);
+  }
+
+  valueFromProps(props) {
+    return props.value !== '' ? props.value : (props.defaultValue || '');
```

Comment:
I just moved it. :) This is how it was and the logic works. I wouldn't want to do `props.value` since that could be the boolean `False`, so it's right to check against `=== ''`